flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/db2/Db2DatabaseSync.java

```diff
@@ -89,12 +89,26 @@ public void registerDriver() throws SQLException {
 
     @Override
     public Connection getConnection() throws SQLException {
+        Properties jdbcProperties = getJdbcProperties();
+        StringBuilder jdbcParams = new StringBuilder();
+        int cnt = 0;
```

Comment:
Is it more appropriate to use boolean here?
```suggestion
       boolean firstParam = true;
```